### DISCUSSION TITLE
images non-visibles en mode apprenant

### DISCUSSION THREAD

**Discussion by Moigaston** [chamilo/chamilo-lms#5590] [General]
Bonjour, 
 j'ai des images dans un cours qui sont visibles en mode prof, mais ne le sont pas en mode apprenant - Chamilo  1.11.26

**Comment by Moigaston** [comment]
Bonjour yverhenne,

Les images en question ne sont pas dans le module "Documents". Je les ai chargées dans l'éditeur de l'introduction.
Le problème que je décris est apparu il y a quelques semaines. Auparavant  tout marchait bien. De plus c'est un problème que je ne rencontre qu'avec le cours dans lesquel ces images se trouvent. Aucun autre cours ne pose ce problème.
Mode apprenant:
![Cours en mode apprenant](https://github.com/chamilo/chamilo-lms/assets/167298740/7bbc5cbb-3a10-4e76-a688-cfc803627164) Mode enseignant:
![Cours en mode enseignant](https://github.com/chamilo/chamilo-lms/assets/167298740/13678a31-137e-40a5-b5e9-e6dd446ee921)

**Comment by Moigaston** [comment]
Salut yverhenne,
Tu peux utiliser des images dans Chamilo sans avoir à les télécharger dans le dossier « Documents » d’un cours mais en allant les chercher  directement sur ton pc avec un éditeur. Et cette manipulation est possible dans tout module de Chamilo où tu as un éditeur : documents, introduction, parcours, exercices, travaux, annonces, et j’en passe. Ça fait des années que je le fais. Les images sont alors sauvegardées dans ton dossier « mes fichiers ». Pas étonnant donc, comme tu l'écris, et si j'ai bien compris, que tu ne trouves pas dans les documents de ton cours les images téléchargées avec un éditeur. 
Mon problème n’est pas de savoir si les images ont été où non téléchargées dans Chamilo, puisqu’elles l'ont été, ni non plus pourquoi elles n'apparaissent pas dans mon cours, puisqu’elles y apparaissent, comme le montrent les captures d’écran que j’ai postées ici. Relis ce que j’ai écrit dans mes deux messages précédents et surtout le titre; tout est là.

**Comment by Coursenligne** [comment]
Le souci décrit ici arrive parfois, notamment quand vous avez 2 serveurs différents (un pour la pré-production et un pour la production).
En effet, les liens sont codés en dur (et donc ne sont pas relatifs).
Un contournement consiste à télécharger les images depuis Documents et de pointer dessus dans le module (dans l'éditeur).
On peut faire beaucoup de choses, mais quand une ne fonctionne pas, ça vaut toujours le coup d'en essayer une autre...
